Formula/libbpg.rb

```diff
@@ -3,6 +3,7 @@ class Libbpg < Formula
   homepage "https://bellard.org/bpg/"
   url "https://bellard.org/bpg/libbpg-0.9.8.tar.gz"
   sha256 "c0788e23bdf1a7d36cb4424ccb2fae4c7789ac94949563c4ad0e2569d3bf0095"
+  license "MIT"
```

Comment:
Might need to check if any extra parts with different licenses are compiled into binaries:
```
10) Licensing
-------------

- libbpg and bpgdec are released under the LGPL license (the FFmpeg
  part is under the LGPL, the BPG specific part is released under the
  BSD license).

- bpgenc is released under the GPL version 2 license. The BPG specific
  code is released under the BSD license. The JCTVC code is released
  under the BSD license. The x265 code is released under the GPL
  version 2 license.

- BPG relies on the HEVC compression technology which may be protected
  by patents in some countries. Most devices already include or will
  include hardware HEVC support, so we suggest to use it if patents
  are an issue.
```

The Monterey bottle at contains:
```
.
├── ChangeLog
├── README
├── bin
│   ├── bpgdec
│   └── bpgenc
└── share
    └── libbpg
        ├── bpgdec.js
        ├── bpgdec8.js
        └── bpgdec8a.js
```